base/src/CCNPPI.cpp

```diff
@@ -3,38 +3,856 @@
 #include "RawImagePlanarMetadata.h"
 #include "Frame.h"
 #include "Logger.h"
-#include "Utils.h"
 #include "AIPExceptions.h"
 #include "CCKernel.h"
-
 #include "npp.h"
 
 class CCNPPI::Detail
 {
 public:
-	Detail(CCNPPIProps &_props) : props(_props)
+	Detail(CCNPPIProps& _props) : props(_props)
 	{
 		nppStreamCtx.hStream = props.stream;
 	}
 
-	~Detail()
+	~Detail() {}
+
+	enum Imageformats
+	{
+		UNSET = 0,
+		MONO = 1,
+		BGR,
+		BGRA,
+		RGB,
+		RGBA,
+		YUV411_I = 10,
+		YUV444,
+		YUV420,
+		UYVY,
+		YUYV,
+		NV12,
+		BAYERBG10 = 20,
+		BAYERBG8,
+		BAYERGB8,
+		BAYERGR8,
+		BAYERRG8
+	};
+
```

Comment:
enumSize = BAYERRG8 + 1 // put comment - last value plus one

Reply:
changed it added enumSize